src/components/Newsletter/Newsletter.js

```diff
@@ -35,50 +42,131 @@ class Newsletter extends React.Component {
+          data-id="2BE4EF332AA2E32596E38B640E905619E90CD5DAC48A878CDEBFFE3B420D8CD24E4AEABAB52A4CE3526219C7A966AE965B84F99C823C89EFF1F01B28DE4F975E"
         >
           <input
             hidden
             readOnly
             aria-label="state"
             value={stateId || ''}
-            id="fieldjlkiyul"
-            maxLength="200"
-            name="cm-f-jlkiyul"
+            id="fieldjrdtwi"
+            maxlength="200"
+            name="cm-f-jrdtwi"
           />
           <input
             hidden
             readOnly
             aria-label="county"
+            id="fieldjrdtwd"
+            maxlength="200"
+            name="cm-f-jrdtwd"
             value={county || ''}
-            id="fieldjlkulhk"
-            maxLength="200"
-            name="cm-f-jlkulhk"
           />
+          <InputHolder>
+            <input
+              type="checkbox"
+              value="wurhhh"
+              id="wurhhh"
+              name="cm-ol-wurhhh"
+            />
+            <label for="checkbox">
+              {' '}
+              Also send me <b>daily news</b> with the latest data and scientific
+              findings on COVID{' '}
+            </label>
+          </InputHolder>
           <input
-            ref={i => (this.emailInput = i)}
-            autoComplete="Email"
-            aria-label="Email"
-            placeholder="Enter your email address"
-            className="js-cm-email-input qa-input-email"
-            id="fieldEmail"
-            maxLength="200"
-            name="cm-wurhhh-wurhhh"
-            required=""
-            type="email"
+            hidden
+            aria-label="alert_list_csv"
+            id="fieldjrdtwy"
+            maxlength="200"
+            name="cm-f-jrdtwy"
+            onChange={value => {
+              console.log(value);
+            }}
+            value={this.state.alertSignUps}
+          />
+          <Autocomplete
+            multiple
+            id="alert-locations"
+            getOptionSelected={(option, value) =>
+              option.full_fips_code === value.full_fips_code
+            }
+            onChange={(event, newValue) => {
+              this.handleSelectChange(newValue);
+            }}
+            options={autocomplete_options}
+            getOptionLabel={option =>
+              option.county
+                ? `${option.county}, ${option.state_code}`
+                : option.state
+            }
+            renderTags={(tagValue, getTagProps) =>
+              tagValue.map((option, index) => (
+                <Chip
+                  label={
+                    option.county
+                      ? `${option.county}, ${option.state_code}`
+                      : option.state
+                  }
+                  {...getTagProps({ index })}
+                />
+              ))
+            }
+            renderInput={params => (
+              <TextField {...params} placeholder="Enter alert locations" />
```

Comment:
FWIW- It's a bit weird that "Enter alert locations" continues to show up even after you have selected 1 or more locations, but I guess it's fine.

![image](https://user-images.githubusercontent.com/206364/84074198-081cb680-a987-11ea-898e-9067480add7d.png)